R_scripts/process_licor_data.R

```diff
@@ -81,7 +81,7 @@ process_licor_data <- function(raw_data, collar_data, plot_data, temp_data) {
            Origin_Elevation = factor(Origin_Elevation, levels = HML),
            Dest_Salinity = factor(Dest_Salinity, levels = HML),
            Dest_Elevation = factor(Dest_Elevation, levels = HML),
-           Date = paste(month(Timestamp), "/", day(Timestamp)),
+           Date = floor_date(Timestamp),
```

Comment:
Can we use `as_data()` here instead? This is what I fixed the problem with yesterday

Reply:
I have no idea what you did yesterday, fix as needed.